pkg/render/node.go

```diff
@@ -43,6 +43,9 @@ var (
 	// The port used by calico/node to report Calico Enterprise internal metrics.
 	// This is separate from the calico/node prometheus metrics port, which is user configurable.
 	nodeReporterPort int32 = 9081
+	// The port used by calico/node to report Calico Enterprise BGP metrics.
+	// This is currently not intended to be user configurable.
+	nodeBGPReporterPort int32 = 9900
```

Comment:
For `nodeReporterPort` there is also an env var set on the node container, do we need the same for this?

Reply:
Good question. 

There is an ENV variable, but the plan for now was to use its defaults. Unless, we think it's better to have a node container ENV in Operator (just for documenting the variable itself), I was going to leave it out until we actually want to expose it to the user in the future.

Reply:
I think leaving it out is a good plan. Ideally we can do that for most things and that way if we add datastore configuration for options we don't need to remove env options to use them.